docs/command_line.rst

```diff
@@ -957,6 +957,34 @@ If you have built a suite called ``churn_model_assumptions`` and a postgres data
 
 This tap can then be run nightly before your model makes churn predictions!
 
+Shell autocompletion for the CLI
+======================
```

Comment:
Nitpick: To make rst happiest the header needs to be at least as long as the text.
```suggestion
=================================
```